server/runtime/src/main/resources/schema/infinispan-server-12.1.xsd

```diff
@@ -422,6 +427,11 @@
                   </xs:documentation>
                </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="security-realm" type="xs:string">
+               <xs:annotation>
+                  <xs:documentation>Names the security realm to use for authentication, cache authorization, and encryption. </xs:documentation>
+               </xs:annotation>
+            </xs:attribute>
```

Comment:
I think we should deprecate the "security-realm" attribute on the HR and REST authentication elements.

Reply:
Yes